tools/README.md

```diff
@@ -36,6 +36,14 @@ Build openpilot with this command:
 scons -u -j$(nproc)
 ```
 
+### Dev Container
+
+openpilot supports VS Code's Dev Containers. Dev containers provide customizable and consistent development environment wrapped inside a container. This means you can develop in a designated environment matching our primary development target, regardless of you local setup.
```

Comment:
It's not purely for vscode: https://containers.dev/supporting

Reply:
Mentioned that. Still focused on VSCode in my short usage instructions since most people will be using that.